src/www/ui/api/documentation/openapi.yaml

```
@@ -1628,6 +1628,52 @@ paths:
                 $ref: '#/components/schemas/Info'
         default:
             $ref: '#/components/responses/defaultResponse'
+  /license/import-csv:
+    post:
+      operationId: importLicense
+      tags:
+        - License
+      summary: Import a csv license
+      description: >
+        Import a csv license to the database
+      requestBody:
+        description: Information about delimiters, inclosure and csv file.
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                inclosure:
+                  type: string
+                  description: Enclosure for string in CSV
+                  default: '"'
+                  required: false
+                delimiter:
+                  type: string
+                  description: Delimiters for fields in CSV
+                  default: ','
+                  required: false
+                file:
```

Content-type `application/json` does not support file uploads. Please revert it back to `multipart/form-data`.
```suggestion
          multipart/form-data:
            schema:
              type: object
              properties:
                enclosure:
                  type: string
                  description: Enclosure for string in CSV
                  default: '"'
                  required: false
                delimiter:
                  type: string
                  description: Delimiters for fields in CSV
                  default: ','
                  required: false
                file_input:
```

---

yeap done